HTML/Layout_1.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+    <style>
+        .text1
```

Comment:
text1, text2, image1, image2, img1_desc, img2_desc are part of the same type of layout called "cards". Each card has a title, a description, and an image that have the same styles. Please redo the stylings.